vcf.c

```diff
@@ -2702,14 +2784,22 @@ static int vcf_parse_format(kstring_t *s, const bcf_hdr_t *h, bcf1_t *v, char *p
         }
         fmt[j].max_l = fmt[j].max_m = fmt[j].max_g = 0;
         fmt[j].key = kh_val(d, k).id;
-        fmt[j].is_gt = !strcmp(t, "GT");
+        fmt[j].is_gt = (t[0] == 'G' && t[1] == 'T');//!strcmp(t, "GT");
```

Comment:
This doesn't check for the NUL at the end, so at the moment it'll match anything with "GT" as a prefix as well as "GT" itself.

Reply:
Oops.  I see I correctly checked in the other places, but missed this one.  Thanks.  Updated